Sources/PointFree/Account/PrivateRss.swift

```diff
@@ -12,6 +12,7 @@ import Syndication
 import Tuple
 
 let accountRssMiddleware = decryptUrlAndFetchUser
+  <<< { trackFeedRequest(userId: \.first.id) >=> $0 }
```

Comment:
This is the core fix. We were tracking the feed request too late in the chain of middleware, and so we would detect the invalid UA and then prevent executing any more of the middleware. This moves it up higher. I should prob refactor this to be a middleware transformer instead of middleware, but can do that later.